src/js/actions/DialogActions.js

```diff
@@ -1,72 +1,84 @@
+var Util = require("../helpers/Util");
 var AppDispatcher = require("../AppDispatcher");
 var DialogEvents = require("../events/DialogEvents");
+var DialogTypes = require("../constants/DialogTypes");
+var dialogScheme = require("../stores/schemes/dialogScheme");
 
 var DialogActions = {
-  alert: function (message, dismissButtonLabel = "OK") {
-    const dialogId = Symbol(message);
-    AppDispatcher.dispatchNext({
-      actionType: DialogEvents.ALERT_SHOW,
-      dialogId: dialogId,
-      dismissButtonLabel: dismissButtonLabel,
-      message: message
-    });
-    return dialogId;
-  },
-  alertDismiss: function (dialogId) {
-    AppDispatcher.dispatchNext({
-      actionType: DialogEvents.ALERT_DISMISS,
-      dialogId: dialogId
-    });
-  },
-  confirm: function (message, successButtonLabel = "OK") {
-    const dialogId = Symbol(message);
+  dialog: function (data) {
+    var dialog = Util.extendObject(dialogScheme, data,
+      {id: Symbol("dialogId")});
```

Comment:
@pierlo-upitup I would stay with Symbol("dialogId"). It's great and verbose if you are using konsole log.